packages/notifications/src/elements/Notification.tsx

```diff
@@ -15,30 +15,22 @@ import { validationIcons, validationHues } from '../utils/icons';
 /**
  * @extends HTMLAttributes<HTMLDivElement>
  */
-export const Notification = forwardRef<HTMLDivElement, INotificationProps>(
-  ({ role, ...props }, ref) => {
-    const Icon = props.type ? validationIcons[props.type] : InfoStrokeIcon;
-    const hue = props.type && validationHues[props.type];
+export const Notification = forwardRef<HTMLDivElement, INotificationProps>((props, ref) => {
+  const Icon = props.type ? validationIcons[props.type] : InfoStrokeIcon;
+  const hue = props.type && validationHues[props.type];
 
-    return (
-      <StyledNotification
-        ref={ref}
-        type={props.type}
-        isFloating
-        {...props}
-        role={role === undefined ? 'status' : role}
-      >
-        {props.type && (
-          <StyledIcon hue={hue}>
-            <Icon />
-          </StyledIcon>
-        )}
+  return (
+    <StyledNotification ref={ref} type={props.type} isFloating {...props} role="alert">
```

Comment:
Please retain the existing `role` override so that consumers can continue to cater to their specific needs.

Reply:
From my testing, any role other than `alert` will not trigger an announcement on JAWS or NVDA. Leaving it up to the user could create an opportunity for an A11Y bug. Can `Notification` be used outside of a "toast" pattern?

Reply:
> Can `Notification` be used outside of a "toast" pattern?

By definition, yes. Conversely, `ToastProvider` can receive other types of content. Let's stick with Garden's approach of a) doing the right thing, b) giving the user control over the corner cases.